### PR TITLE
Index rephrasing + restructuring

### PR DESCRIPTION
**Pull request by Tzky**
This PR aims to:

* improve the readability of the Docs for novice users
* clean up the index
* move the structure page to the ``about`` section

### CODE REVIEW

**Comment by Spongy** [comment]
Setting up PR reference, please wait...

File: source/about/faq.rst [hunk 1]
```diff
@@ -46,7 +46,7 @@ Where do I get Plugins for Sponge?
 ----------------------------------
 
 On our official plugin repository, which is called Ore. Development has already started on GitHub. Temporarily, plugins
-are hosted on the SpongeForums.
+are hosted on the `SpongeForums <https://forums.spongepowered.org/c/plugins>`_.
```
Comment (12AwsomeMan34) [inline source/about/faq.rst]: Maybe /plugins/plugin-releases ? /plugins can include a lot of development stuff.

File: source/index.rst [hunk 2]
```diff
@@ -7,12 +7,21 @@ Introduction
 
 Welcome to SpongeDocs, the official documentation for the open-source `Sponge <http://spongepowered.org>`__ project.
 
-The Sponge project currently has four main components:
+The SpongeAPI is your all new plugin API for Minecraft. You can run it on a plain vanilla server (as SpongeVanilla) or
+on your fully modded MinecraftForge server (as SpongeForge). Grab your your copy :doc:`here <downloads>`.
```
Comment (12AwsomeMan34) [inline source/index.rst]: your your?

File: source/server/getting-started/implementations/spongeforge.rst [hunk 3]
```diff
@@ -50,6 +50,80 @@ The file name ``spongeforge-1.8-1577-3.0.0-BETA-1000.jar`` is compatible with mi
...
+   sure to use **exactly the same build number** as :doc:`the one listed in the filename of the Sponge download
+   <spongeforge>`.
+#. Run the provided Forge installer. A new Forge profile will be created in the Minecraft launcher.
+#. Open the Minecraft launcher, and select the new Forge profile.
+#. Click "Options" and click "Open Game Dir".
+#. Download SpongeForge from the Sponge website and put it into the ``mods`` folder. Create the folder if it does
+   not yet exist.
+#. Sponge should work in both in single player and if you open your world to LAN.
+
+Next, learn how you can create and use a :doc:`launch-script <../launch-script>`, how to
```
Comment (12AwsomeMan34) [inline source/server/getting-started/implementations/spongeforge.rst]: A launch script for a single-player world? Correct me if I'm wrong, but I believe that something like this belongs under Dedicated Servers.

Comment (Tzky) [inline source/server/getting-started/implementations/spongeforge.rst]: I'm not even sure how that one got there... Fixing now.

File: source/about/faq.rst (hunk 1 above)
Comment (Tzky) [inline source/about/faq.rst]: Agreed and already changed :)

**Comment by Tzky** [comment]
merged in 2331251

**Pull request closed without merging**